lib/Slider/Slider.js

```diff
@@ -898,6 +898,7 @@ module.exports = kind(
 	*/
 	previous: function () {
 		this.set('value', this.value - this._jumpIncrementAmount);
+		if (this.popup && this.$.popup.getShowing()) return true;
```

Comment:
Holding matter makes sense. 
I think to keep consistency, behavior of pointer mode need to be same as 5-way mode.

Let ask it to Robert.G and @Djspaceg